modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/info/item/provider/test/DDMFormValuesInfoFieldValuesProviderTest.java

```diff
@@ -105,6 +122,44 @@ public void testGetInfoFieldValuesCheckboxMultipleDDMFormFieldType()
 				).build()));
 	}
 
+	@Test
+	public void testGetInfoFieldValuesNestedStructure() throws Exception {
```

Comment:
Could you rename to `testGetInfoFieldValuesFieldSetDDMFormFieldType`? This is to be in line with the names of the other methods.
What do you think?

Tip: sometimes we call "nested fields," but it is best to follow the name defined in the code in the tests. This type of field is named "fieldset."

Reply:
It makes a lot of sense!!